apps/aeutils/src/aeu_mtrees.erl

```diff
@@ -102,6 +103,18 @@ new_with_backend(empty, DB) ->
 new_with_backend(<<_:256>> = Hash, DB) ->
     aeu_mp_trees:new(Hash, DB).
 
+-spec gc_old_nodes(mtree()) -> mtree().
+gc_old_nodes(Tree) ->
+    %% TODO: this is a workarond and not a proper GC
```

Comment:
This just re-builds the tree?! But that will skip all things no longer accessible I guess

Reply:
Yes. A proper GC will be implemented.